models/reviews.models.js

```diff
@@ -1,20 +1,52 @@
 const db = require("../db/connection.js");
 const { checkReviewExists } = require("../db/seeds/utils.js");
 
-exports.selectReviews = () => {
-  return db
-    .query(
-      `
-      SELECT reviews.owner, title, reviews.review_id, category, review_img_url, reviews.created_at, reviews.votes, reviews.designer, COUNT(comments.review_id) AS comment_count
-      FROM reviews 
-      LEFT JOIN comments ON reviews.review_id = comments.review_id
-      GROUP BY reviews.review_id
-      ORDER BY created_at DESC;
-      `
-    )
-    .then((response) => {
-      return response.rows;
-    });
+exports.selectReviews = (category, sort_by = "created_at", order = "desc") => {
+  const validColumns = [
+    "review_id",
+    "title",
+    "review_body",
+    "designer",
+    "review_img_url",
+    "votes",
+    "category",
+    "owner",
+    "created_at",
+    "comment_count",
+  ];
+
+  const validCategories = ["euro game", "dexterity", "social deduction"];
```

Comment:
Be careful! You don't want to have to update this every time a new category is added. How else could you verify if a category is valid or not?